### PR TITLE
Add person uid

### PR DESCRIPTION
**Pull request by joshualiangxy**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -71,7 +86,8 @@ public boolean isSamePerson(Person otherPerson) {
 
         return otherPerson != null
                 && otherPerson.getName().equals(getName())
-                && (otherPerson.getPhone().equals(getPhone()) || otherPerson.getEmail().equals(getEmail()));
+                && (otherPerson.getPhone().equals(getPhone())
+                || otherPerson.getEmail().equals(getEmail()));
```
Comment (jazerler) [inline src/main/java/seedu/address/model/person/Person.java:90]: Is this missing out the UID of otherPerson for comparing if both Persons are the same?

**Review by jazerler — COMMENTED** [review]
Just need to confirm the comment on isSamePerson in Person.java, otherwise it's good to merge.

File: src/main/java/seedu/address/model/person/Person.java (hunk 1 above)
Comment (joshualiangxy) [reply src/main/java/seedu/address/model/person/Person.java:90]: This part checks for if you are trying to add a new person with the same name and phone or same name and email, so the UID can be different here.

**Review by jazerler — APPROVED** [review]
isSamePerson could be named better, but the functionality is likely to be changed in future so it should be good to merge for now.

**Pull request merged**